packages/ui/src/views/webhooks/webhook-list/types.ts

```diff
@@ -0,0 +1,67 @@
+import { TranslationStore } from '@views/repo'
+
+export interface WebhookType {
+  id: number
+  name: string
+  enabled: boolean
+  createdAt: string
+  description: string
+}
+
+export interface WebhookStore {
+  webhooks: WebhookType[] | null
+  // onDelete: (id: number) => void
+  error: string | undefined
+  setError: (error?: string) => void
+  totalPages: number
+  webhookLoading: boolean
+  setWebhookLoading: (webhookLoading: boolean) => void
+  page: number
+  setPage: (page: number) => void
+  setWebhooks: (data: ListRepoWebhooksOkResponse, headers: Headers | undefined) => void
+}
+export declare type ListRepoWebhooksOkResponse = OpenapiWebhookType[]
+
+export interface OpenapiWebhookType {
+  created?: number
+  created_by?: number
+  description?: string
+  display_name?: string
+  enabled?: boolean
+  has_secret?: boolean
+  id?: number
+  identifier?: string
+  insecure?: boolean
+  latest_execution_result?: EnumWebhookExecutionResult
+  parent_id?: number
+  parent_type?: EnumWebhookParent
+  triggers?: EnumWebhookTrigger[] | null
+  updated?: number
+  url?: string
+  version?: number
+}
+
+export declare type EnumWebhookParent = 'repo' | 'space'
+
+export declare type EnumWebhookExecutionResult = 'fatal_error' | 'retriable_error' | 'success' | null
+
+export declare type EnumWebhookTrigger =
```

Comment:
Can we change the type to enum?

Reply:
it would make sense to keep it as this since its just from api